File: testers/grpc/scala/generated-and-checked-in/com/example/grpc/EchoService.scala
package com.example.grpc



/** Clean service interface for EchoService gRPC service */
trait EchoService {
  def echoScalarTypes(request: ScalarTypes): ScalarTypes

  def echoCustomer(request: Customer): Customer

  def echoOrder(request: Order): Order

  def echoInventory(request: Inventory): Inventory

  def echoOuter(request: Outer): Outer

  def echoOptionalFields(request: OptionalFields): OptionalFields

  def echoWellKnownTypes(request: WellKnownTypesMessage): WellKnownTypesMessage

  def echoPaymentMethod(request: PaymentMethod): PaymentMethod

  def echoNotification(request: Notification): Notification
}

File: testers/grpc/kotlin/generated-and-checked-in/com/example/grpc/EchoService.kt
package com.example.grpc



/** Clean service interface for EchoService gRPC service */
interface EchoService {
  abstract fun echoCustomer(request: Customer): Customer

  abstract fun echoInventory(request: Inventory): Inventory

  abstract fun echoNotification(request: Notification): Notification

  abstract fun echoOptionalFields(request: OptionalFields): OptionalFields

  abstract fun echoOrder(request: Order): Order

  abstract fun echoOuter(request: Outer): Outer

  abstract fun echoPaymentMethod(request: PaymentMethod): PaymentMethod

  abstract fun echoScalarTypes(request: ScalarTypes): ScalarTypes

  abstract fun echoWellKnownTypes(request: WellKnownTypesMessage): WellKnownTypesMessage
}